.docker/Dockerfile

```diff
@@ -13,4 +13,20 @@ RUN chown -R www-data:www-data /var/www/.npm
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
 RUN apt install -y nodejs
 
+RUN pecl channel-update pecl.php.net \
```

Comment:
From the docker page:
Some extensions are not provided with the PHP source, but are instead available through PECL. To install a PECL extension, use pecl install to download and compile it, then use docker-php-ext-enable to enable it:

```
FROM php:7.4-cli
RUN pecl install redis-5.1.1 \
    && pecl install xdebug-2.8.1 \
    && docker-php-ext-enable redis xdebug
```

You don't have to do the `yes`.

It's also better to use one RUN per line if possible instead of using `&&`.
If you want to copy a file, it will be easier and better :)

Reply:
But wouldn't every RUN add an additional layer to the image? In this specific case i think having a RUN per line would be a bit too much.